instrumentation/netty/netty-4.1/library/src/main/java/io/opentelemetry/instrumentation/netty/v4_1/NettyClientTelemetryBuilder.java

```diff
@@ -114,7 +114,7 @@ public NettyClientTelemetry build() {
                 "io.opentelemetry.netty-4.1",
                 NettyConnectionInstrumentationFlag.DISABLED,
                 NettyConnectionInstrumentationFlag.DISABLED,
-                Collections.emptyMap(),
+                CommonConfig.get().getPeerServiceResolver(),
```

Comment:
Hi @mateuszrzeszutek ,

`CommonConfig.get().getPeerServiceResolver()`

Should have been called here but it cannot be found here.
I don't know much about gradle .

Can you help here ?